src/main/java/ru/hogwarts/school/service/StudentService.java

```diff
@@ -100,39 +98,48 @@ public double avgAgeStudent() {
 
     public void getStudentsFromTread() {
         List<Student> allStudents = studentRepository.findAll();
-        System.out.println(allStudents.get(0));
-        System.out.println(allStudents.get(1));
+
+        String info1 = allStudents.get(0) + " ; " + allStudents.get(1);
+        logger.info(info1);
+
         Thread thread1 = new Thread(() -> {
-            System.out.println(allStudents.get(2));
-            System.out.println(allStudents.get(3));
+            String info2 = allStudents.get(2) + " ; " + allStudents.get(3);
+            logger.info(info2);
         });
         Thread thread2 = new Thread(() -> {
-            System.out.println(allStudents.get(4));
-            System.out.println(allStudents.get(5));
+            String info3 = allStudents.get(4) + " ; " + allStudents.get(5);
+            logger.info(info3);
         });
 
         thread1.start();
         thread2.start();
 
 
     }
+    static private int count = 0;
 
 
-    private synchronized void printStudent(Student student) {
-        System.out.println(student);
+    private synchronized String printStudent() {
+        List<Student> allStudents = studentRepository.findAll();
+        String st= String.valueOf(allStudents.get(count));
+        count++;
```

Comment:
```suggestion
        count = count % 6;
```
иначе будет выход за границы